index.html

```diff
@@ -13,42 +13,135 @@
     />
     <meta name="author" content="LayoutIt!" />
 
+    <style type="text/css">
+
+      /*set the background*/
+      .bg {
+            background: url('https://jlprojects.com/wp-content/uploads/2017/06/Sea-Ranch-Boca-Raton-Condo-Remodeling-3.jpg');
```

Comment:
This image would be better downloaded and put in a folder named "img", and use the link "/img/someimg.jpg". Some websites forbid external links.